sourced/ml/transformers/coocc.py

```diff
@@ -0,0 +1,87 @@
+import itertools
+import operator
+
+from scipy import sparse
+from bblfsh import role_id
+
+from sourced.ml.models import Cooccurrences
+from sourced.ml.transformers import Transformer
+
+
+class CooccModelSaver(Transformer):
+    def __init__(self, output, tokens_list, **kwargs):
+        super().__init__(**kwargs)
+        self.tokens_list = tokens_list
+        self.output = output
+
+    def __call__(self, sparce_matrix):
```

Comment:
Please annotate `sparse_matrix` wit ha type hint and a docstring. And btw there is a misprint: spar**c**e